src/services/scope/uc/index.js

```diff
@@ -1,7 +1,5 @@
-const ScopeUc = require('./scope.uc');
-const disallow = require('../../../common/disallow.hook');
+const ScopeBcFacade = require('./scope.facade');
 
 module.exports = function setupScopeUc(app) {
-	app.use('/scopeUc', new ScopeUc());
-	app.service('scopeUc').hooks(disallow);
+	app.registerFacade('scope', new ScopeBcFacade());
```

Comment:
why separated in additional function?
app.facade('scope') as getter and app.facade('scope', new ScopeBcFacade()) as setter is more feather like logic and give you in some situation a better handling options.

Reply:
app.facade() to list all facades is also nice.

Reply:
I dont think you can set feathers services with app.service(), you do that with app.use().
Even if, I much prefer this less error-prone way... I wouldnt want anyone to confuse those two functions.

I would be interested in where you think that would give us better handling options, as I cant come up with any?